sycl/source/handler.cpp

```diff
@@ -1093,7 +1093,7 @@ void handler::extractArgsAndReqs() {
   if (impl->MKernelData.getDeviceKernelInfoPtr() == nullptr) {
     impl->MKernelData.setDeviceKernelInfoPtr(
         &detail::ProgramManager::getInstance().getOrCreateDeviceKernelInfo(
-            toKernelNameStrT(getKernelName())));
+            std::string(MKernel->getName())));
```

Comment:
`toKernelNameStrT` implementation is under preview, I'd prefer not touching its uses and leave the actual "simplification"/inlining of `*KernelNameStr*T` types to after all the non-preview uses are completely removed.

Reply:
> `toKernelNameStrT` implementation is under preview,

It can't be under preview because the code I'm changing is *not* under preview. I made the change because local build failed, but I've just realized that `detail::` is missing